packages/app/src/HomePage.utils.ts

```diff
@@ -118,6 +118,24 @@ export function canCreate(resourceType: string): boolean {
   return resourceType !== 'Bot' && resourceType !== 'ClientApplication';
 }
 
+export function getNewResourceEndpoint(resourceType: string, isProjectAdmin: boolean): string {
```

Comment:
I didn't dig much into the codebase, but could this benefit from better typing?

```suggestion
export function getNewResourceEndpoint(resourceType: ResourceType, isProjectAdmin: boolean): string {
```

The same goes with `canCreate`.